docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,269 @@
-[[get-started-docker]]
-== Running the {stack} on Docker
+[[get-started-stack-docker]]
+== Running the {stack} ("ELK") on Docker
+++++
+<titleabbrev>Running the {stack} on Docker</titleabbrev>
+++++
 
 The Elastic Docker registry contains Docker images for all the products in
 the {stack}: https://www.docker.elastic.co/.
 
-[float]
-[[run-stack-docker]]
-=== Run with Docker Compose
+You can <<run-docker-secure,start the {stack} in Docker>> with security enabled
+and configured by default. This option is great for quickly getting started with
+{es} and {kib} for testing and development.
+
+You can also <<get-started-docker-tls,start the {stack} with Docker Compose>> to
+create a secured, multi-node cluster with a connected {kib} instance. This
+option requires more manual steps, but results in a more robust cluster.
```

Comment:
Should we qualify what we mean with "robust cluster" here ? It reads slightly generic

Reply:
Yeah, I think that we can expand a bit more without getting too in depth. How about:
>This option requires more manual steps, but results in a more resilient cluster with greater capacity and reliability.